Pear/View/LocationFooterView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  LocationFooterView.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 3/21/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class LocationFooterView: UICollectionReusableView {
+
+    private let label = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        label.font = ._12CircularStdBook
+        label.numberOfLines = 0
+        label.textColor = .greenGray
+        addSubview(label)
+
+        label.snp.makeConstraints { make in
+            make.leading.trailing.top.bottom.equalToSuperview().inset(5)
```

Comment:
I don't remember, but can you say 

```suggestion
            make.equalToSuperview().inset(5)
```

Reply:
Pretty sure you can